HIP/hip-830.md

```diff
@@ -0,0 +1,343 @@
+---
+hip: 830
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Last Call
+last-call-date-time: 2023-12-04T07:00:00Z
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: 2023-11-30
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the `roundBirth` to the metadata of new events. The `roundBirth` value is 1 + the latest round to have come
+   to consensus at the time the event was created. This values is used to look up the correct consensus
+   roster for validating the event's signature on other nodes.
+2. Represent references to events as `EventDescriptors`. EventDescriptors are used by an event to link
```

Comment:
Perhaps clearer terminology here would be something like,
"Combine existing data about each parent into a data structure (hash, creator, generation), and add the parent's birthRound to that data structure"